fleetspeak/src/server/components/proto/fleetspeak_components/config.proto

```diff
@@ -57,6 +57,11 @@ message Config {
   // with this server's notification port. Has no effect when
   // notification_listen_address is unset.
   string notification_public_address = 6;
+
```

Comment:
The comment for the `https_config` field above says "Required". Please update.

Reply:
Done.